pkg/exchange/okex/query_kline_test.go

```diff
@@ -0,0 +1,74 @@
+package okex
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryKlines(t *testing.T) {
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+	}
+
+	e := New(key, secret, passphrase)
+
+	queryOrder := types.OrderQuery{
+		Symbol: "BTC-USDT",
+	}
+
+	now := time.Now()
+	// test supported interval - minute
+	klineDetail, err := e.QueryKLines(context.Background(), queryOrder.Symbol, types.Interval("1m"), types.KLineQueryOptions{
```

Comment:
Why you pass customizing variable 1m?
Using types.Interval1M is preferable to customizing the interval.